src/Razor/test/Microsoft.AspNetCore.Razor.OmniSharpPlugin.Test/DefaultProjectChangePublisherTest.cs

```diff
@@ -21,12 +22,14 @@ public DefaultProjectChangePublisherTest(ITestOutputHelper testOutput)
     }
 
     [Theory]
-    [InlineData(OmniSharpProjectChangeKind.DocumentAdded)]
-    [InlineData(OmniSharpProjectChangeKind.DocumentRemoved)]
-    [InlineData(OmniSharpProjectChangeKind.ProjectChanged)]
-    public async Task ProjectManager_Changed_EnqueuesPublish(OmniSharpProjectChangeKind changeKind)
+    [InlineData(nameof(OmniSharpProjectChangeKind.DocumentAdded))]
+    [InlineData(nameof(OmniSharpProjectChangeKind.DocumentRemoved))]
+    [InlineData(nameof(OmniSharpProjectChangeKind.ProjectChanged))]
+    public async Task ProjectManager_Changed_EnqueuesPublish(string changeKindStr)
```

Comment:
I think the VS Code tests were actually disabled in CI a while ago, lol might be good to eventually verify they still work/pass 😅 (prob can be done in a different PR)